pgcommitfest/commitfest/migrations/0013_no_patches_in_future_cfs.py

```diff
@@ -0,0 +1,62 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("commitfest", "0012_add_parked_cf_status"),
+    ]
+    operations = [
+        migrations.RunSQL(
+            """
+CREATE FUNCTION assert_poc_not_future_for_poc()
```

Comment:
As also said on the previous PR: Instead of messing around with triggers. We should simply remove the concept of future commitfests.